variables.tf

```diff
@@ -411,3 +411,9 @@ variable "ca_cert_identifier" {
   type        = string
   default     = null
 }
+
+variable "enabled_egress" {
```

Comment:
We have a convention to name variables `*_enabled`. Can you update this to `egress_enabled`?

Reply:
Yep, thought I'd done that before but I got it backwards. oops